modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -620,7 +830,15 @@ private CompletableFuture<Boolean> stopReplicaInternal(ReplicationGroupId replic
             }
         });
 
-        return isRemovedFuture;
+        return isRemovedFuture
+                .thenApply(v -> {
+                    try {
+                        raftManager.stopRaftNodes(replicaGrpId);
```

Comment:
Shouldn't it be a part of concrete Replica stop?

Reply:
This method `stopRaftNodes` is `raftManager` specific. `RaftManager` is responsible for nodes' starting and stopping => we're starting them in `ReplicaManager` => we should stop them in `ReplicaManager` too with `RaftManager` as well.

Reply:
Ok, let's do it in this way now. In future, start and stop should be placed under Replica itself

Reply:
Added TODO with https://issues.apache.org/jira/browse/IGNITE-22373 in `replicaMgr` usages, start/stop are included